package.json

```diff
@@ -34,6 +34,7 @@
     "start:dev": "concurrently 'yarn start:dev:relay' 'yarn start:dev:server'",
     "start:dev:server": "NODE_ENV=development node --inspect --max_old_space_size=3072 -r dotenv/config ./src/index.dev.js",
     "start:dev:relay": "NODE_ENV=development yarn relay --watch",
+    "start:dev:debug": "node --inspect src/index.dev.js",
```

Comment:
Intentional change for this pr? seems unrelated.

Reply:
It was intentional, I will move it to another PR because I already have 😄